app/controllers/application_controller/ci_processing.rb

```diff
@@ -1868,6 +1868,7 @@ def process_cloud_object_storage_buttons(pressed)
     task = pressed.sub("#{klass.name.underscore.to_sym}_", "")
 
     return tag(klass) if task == "tag"
+    return javascript_redirect(:controller => klass.name.underscore.to_sym, :action => "new") if task == "new"
 
```

Comment:
I did some research and now I'm confused: should "Add CloudObjectStoreContainer" option appear only next to the list of all containers and be hidden next to the list of containers per Manager?

In other words, should this be hidden:
![capture](https://cloud.githubusercontent.com/assets/8102426/24291689/4ac28da0-108a-11e7-854f-79397c1e64c9.PNG)
?